### DISCUSSION TITLE
Tftproot & webroot directory location

### DISCUSSION THREAD

**Discussion by marbaa** [cobbler/cobbler#2916] [Version specific questions]
I'm playing for two days with cobbler 3.x (I must say, it is a lot of pain with 3.x :-( ) and figuring if there is benefit to upgrade it from 2.8. Is this new, design choice or something is not wokring at the background? :-) I Mean the ```@@xxx@@``` folders. I thought that it is some kind of variable in ```settings.yaml``` which cobbler will automagically expand it to /var/lib/cobbler.

![image](https://user-images.githubusercontent.com/40859847/150989247-e56caeb9-df7b-4eb9-9f22-3d34d0fe883f.png)

**Comment by SchoolGuy** [comment]
@marbaa Those folders are replaced at buildtime and provide reasonable defaults per distro. The system is tested and should work. If not then I would be interested in fixing the specifc case in which it doesn't work.

**Reply by marbaa** [reply]
Good, above picture is from minimal installation of Fedora 35. Cobbler installed from repository ```dnf install cobbler``` pasted config from branch 3.2.2 from here source and started cobblerd.  @@ folders were created.

```/tftpboot``` folder was created by installing syslinux-tftpboot package.

Tested same steps on Centos8 Stream, I think in that case I manually replaced the variable in yaml file with /var/lig/cobbler path, to get rid of the message from ```cobbler check``` command.

Do you want more details?

**Reply by SchoolGuy** [reply]
Yeah in our repos we obviously don't have those folders in clear text because this is from source. So that was to be expected then.

See https://github.com/cobbler/cobbler/blob/release32/config/cobbler/settings.yaml#L241 for example.

**Reply by marbaa** [reply]
Is there some debug output I can provide? Or anything else?

Maybe I will try to create som VM with openSuse.
